### ISSUE TITLE
Cannot add bot to channel

### ISSUE THREAD

**Issue by abundov** [NullPxl/NullCTF#14]
![Screenshot_20201125-013553_Firefox](https://user-images.githubusercontent.com/49936643/100196705-dfe62d00-2ebe-11eb-9c7f-1665c149d62a.jpg)

This is my issue. Please help I want to add this to my server but the bot is not verified it seems. 

Thank you.

**Comment by NullPxl** [comment]
From the readme:

> As of October 7th 2020 the bot can no longer be invited to new servers.
This is due to Discord's new verification requirements for bots in over 100 servers. It would require me to verify my identity with photo id, something which I am not personally comfortable with. I understand that this isn't exactly convenient, but I hope that by making this code open source (and including setup instructions at the bottom of this readme) people can still use this bot.

In other words: discord is forcing bot creators to throw away privacy for the privilege of having popular bots.

When new instances of the bot are put up I will add the invite links in the readme, as well as in the [support server](https://discord.gg/yf8E2s8) 

Sorry for the delayed response, usually I get an email when new issues are opened but didn't this time for some reason.

**Comment by NullPxl** [comment]
New instance is up, let me know if there's any issues! 
https://discord.com/api/oauth2/authorize?client_id=803843628241387530&permissions=268561520&scope=bot

**Comment by abundov** [comment]
Thank you for addressing this! Just got the bot to my channel.
